<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageSettings</class>
 <widget class="QWidget" name="PageSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="objects">
      <attribute name="title">
       <string>Elements</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="addTextButton">
             <property name="text">
              <string>Add Text</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addGifButton">
             <property name="text">
              <string>Add Image</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="empty">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>NOTHING IS SELECTED</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="text">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Font:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="fontsCombo">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Color:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="colorBtn">
                 <property name="styleSheet">
                  <string notr="true">background-color: black</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Font size:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="size0Button">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">fontSizeGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="size1Button">
                 <property name="text">
                  <string>1</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">fontSizeGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="size2Button">
                 <property name="text">
                  <string>2</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">fontSizeGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="boldButton">
                 <property name="text">
                  <string>Bold</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Alignment:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="alignmentComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTextEdit" name="textEdit"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Animation:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="animationComboBox"/>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Speed:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="speedSpinBox">
                 <property name="maximum">
                  <number>60</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Fade colour:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="colorFadeBtn">
                 <property name="styleSheet">
                  <string notr="true">background-color: black</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Speed:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="fadeSpeedSpinBox"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="gif">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>GIF GIF GIF</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="elements">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>350</width>
            <height>435</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Elements</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QListWidget" name="elementsList"/>
           </item>
           <item>
            <widget class="QLineEdit" name="elementName"/>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QPushButton" name="duplicateBtn">
                <property name="text">
                 <string>Duplicate</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="moveUp">
                <property name="text">
                 <string>Move Up</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="moveToTop">
                <property name="text">
                 <string>Move To Top</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="deleteBtn">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="moveDown">
                <property name="text">
                 <string>Move Down</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPushButton" name="moveToBottom">
                <property name="text">
                 <string>Move To Bottom</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="events">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>350</width>
            <height>435</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Events</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <attribute name="title">
       <string>Metadata</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="fontSizeGroup"/>
 </buttongroups>
</ui>
